### PR TITLE
docs(operator): Operator documentation for docs app.

### PR DESCRIPTION
**Pull request by Malvolio**
**Description:**

Adding a section to the guide with a brief description of Operators.

**Related issue (if exists):**

### CODE REVIEW

File: docs_app/content/guide/operators.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# Operators
+
+An operator is just a function that takes an Observable as its input and returns another Observable.  For example, there is an operator called `map()`, that is equivalent to the Array method of the same name. Just as `[1, 2, 3].map(x => x * x)` will yield `[1, 4, 9]`, the Observable created like this:
```
Comment (JWO719) [inline docs_app/content/guide/operators.md]: could you add links to to the api docs of the mentioned operators

File: docs_app/content/guide/operators.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+```
+
+will emit `1`, `4`, `9`.  Another useful operator is `first()`:
+
+```ts
+import { of } from 'rxjs';
+import { first } from 'rxjs/operators';
+
+first()(of(1, 2, 3).subscribe((v) => console.log(`value: ${v}`));
```
Comment (JWO719) [inline docs_app/content/guide/operators.md]: here's a missing bracket 
`first()(of(1, 2, 3))`

File: docs_app/content/guide/operators.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
+# Operators
+
+An operator is just a function that takes an Observable as its input and returns another Observable.  For example, there is an operator called `map()`, that is equivalent to the Array method of the same name. Just as `[1, 2, 3].map(x => x * x)` will yield `[1, 4, 9]`, the Observable created like this:
+
+```ts
+import { of } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+map(x => x * x)(of(1, 2, 3).subscribe((v) => console.log(`value: ${v}`));
```
Comment (JWO719) [inline docs_app/content/guide/operators.md]: here's a missing bracket 
`map(x => x * x)(of(1, 2, 3))`

File: docs_app/content/guide/operators.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+## Creating custom observables
+
+### Use the `pipe()` function to make new operators
+
+If there is a commonly used sequence of operators in your code, use the `pipe()` function to extract the sequence into a new operator. Even if a sequence is not that common, breaking it out into a single operator can improve readability.
+
+For example, you could make a function that discarded odd values and doubled even values like this:
+
+```ts
+function discardOddDisableEven() {
```
Comment (JWO719) [inline docs_app/content/guide/operators.md]: could you maybe add the import for filter, map and pipe

File: docs_app/content/guide/operators.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+  return pipe(
+    filter(v => ! (v % 2)),
+    map(v => v + v),
+  );
+}
+```
+
+(The `pipe()` function is analogous to, but not the same thing as, the `.pipe()` method on an Observable.)
+
+It is more complicated, but if you have to write an operator that cannot be made from a combination of existing operators (a rare occurrance), you can write an operator from scratch, like this:
```
Comment (JWO719) [inline docs_app/content/guide/operators.md]: I would like having here another section, e.g. :

### create from scratch

File: docs_app/content/guide/operators.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+
+
+```ts
+function delay(delayInMillis) {
```
Comment (JWO719) [inline docs_app/content/guide/operators.md]: it would be awesome if you could add the missing imports

**Comment by JWO719** [comment]
@Malvolio can I support you in any way? I would really like to merge this one. Let me know if there is anything I can support you with!

File: docs_app/content/guide/operators.md (hunk 1 above)
Comment (Malvolio) [reply docs_app/content/guide/operators.md]: Done

File: docs_app/content/guide/operators.md (hunk 3 above)
Comment (Malvolio) [reply docs_app/content/guide/operators.md]: done

File: docs_app/content/guide/operators.md (hunk 5 above)
Comment (Malvolio) [reply docs_app/content/guide/operators.md]: done

File: docs_app/content/guide/operators.md (hunk 6 above)
Comment (Malvolio) [reply docs_app/content/guide/operators.md]: done

**Comment by Malvolio** [comment]
@JWO719 -- did a merge with the existing document and added in your changes.  Let me know what you think.

File: docs_app/content/guide/operators.md (hunk 4 above)
Comment (JWO719) [reply docs_app/content/guide/operators.md]: I think this is the last missing bit and afterwards I happily merge this one! :D

Comment (Malvolio) [reply docs_app/content/guide/operators.md]: Done (and corrected the function name)

**Pull request merged**